include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -24,109 +24,58 @@
 #include <seqan3/alphabet/quality/concept.hpp>
 #include <seqan3/core/concept/core_language.hpp>
 #include <seqan3/core/concept/tuple.hpp>
+#include <seqan3/core/detail/debug_stream_type.hpp>
 #include <seqan3/core/detail/int_types.hpp>
 #include <seqan3/core/type_traits/pack.hpp>
 #include <seqan3/core/type_traits/transformation_trait_or.hpp>
 #include <seqan3/core/tuple_utility.hpp>
 #include <seqan3/std/concepts>
 
+
 namespace seqan3::detail
 {
 
-/*!\brief Evaluates to true if one of the components of seqan3::alphabet_tuple_base satisifes a compile-time
- *        predicate.
- * \tparam tuple_t         A specialisation of seqan3::alphabet_tuple_base.
- * \tparam tuple_derived_t The CRTP derived type of `tuple_t`.
- * \tparam fun_t               A template template that takes target_t as argument and exposes an `invoke` member type
- *                             that evaluates some predicate and returns `std::true_type` or `std::false_type`.
- * \tparam target_t            The type you wish to query.
- * \ingroup composite
- *
- * \details
- *
- * To prevent recursive template and/or concept instantiation this call needs to be guarded against many exceptions.
- * See the source file for more details.
- */
-
-// anchor is false
-template <typename tuple_t, typename tuple_derived_t, template <typename> typename fun_t, typename other_t>
-inline bool constexpr one_component_is = false;
-
-//!\cond
+#if SEQAN3_WORKAROUND_GCC_LAZY_REQUIRES
+// mechanism to prevent triggering GCC bugs in comparison of alphabet tuples with their components.
```

Comment:
Is this a reported issue?
if yes, you could add the link

Reply:
No, and since GCC7 doesn't receive fixes any more, there is no point.